llsd/serde_xml.py

```diff
@@ -92,18 +101,17 @@ def DATE(self, v):
     def ARRAY(self, v):
         return self._elt(
             b'array',
-            b''.join([self._generate(item) for item in v]))
+            lambda: [self._generate(item) for item in v])
```

Comment:
As with the previous class, these will always be returning None (return value of _elt()), yes?

Reply:
Yes. I left the `return` statements in place because they're not wrong, albeit slightly misleading now.